### DISCUSSION TITLE
Push notifications for the app

### DISCUSSION THREAD

**Discussion by anicolao** [Ethrel/NPA-Mobile#12] [Ideas]
I'd love to make it either the default or a checkbox for users to enable push notifications on your app. I already have server side support for notifications, I get them via `pushover`, but it would be so much nicer to just get them by default.

I haven't really done notifications for Android before, I'd probably go with using Firebase's push notifications with the goal of having them work across all platforms (desktop browser, android, iOS). Thoughts? Are you willing to work together to make it happen, or willing to take pull requests to make it happen?

**Comment by Ethrel** [comment]
I'm defenitely amenable to the idea and accepting PRs for that. It's actually one of the longer-term ideas that I was going to look into (functionality first, so the hotkey buttons I'm playing with are a higher priority).

It does indeed look like firebase would be the way to go.

If you do up a pr, make sure to pr onto development please :)

As part of the better hotkey bar and ui I'm in the middle of working on, I do plan on setting up a settings screen of some kind, so a toggle (or multiple toggles for varying different notifications) could go in there for it once that's done. Most likely, in the meantime, it'd be a default all-on setting.
